gc/base/Forge.hpp

```diff
@@ -67,8 +67,7 @@ friend class ::MM_GCExtensionsBase;
 	 * 
 	 */
 	void tearDown();
-	
-public:
+
```

Comment:
Did you do this so you could write the tests? It looks reasonable, I'm just wondering why. You should also get rid of the friend declaration above.

Reply:
Yes for tests, friend class removed